src/translate/index.ts

```diff
@@ -127,9 +128,11 @@ export default async function translate(this: Command) {
       continue;
     }
 
+    let contextBuffer = "";
+
     progress.start(filteredTranslations.length, 0);
     for (let [key, sourceText] of filteredTranslations) {
-      if (options.compiledI18n && Boolean(sourceText) === false) {
+      if (options.compiledI18n && !Boolean(sourceText)) {
```

Comment:
@genox please revert this. The upper one is far more readable.
You can easily oversee the exclamation mark.